build-prybar.nix

```
@@ -49,6 +49,8 @@ buildGoModule {
     # is running in.
     export HOME=$(echo pwd)
 
+    patchShebangs ./tests/${language}/*.exp
+    patchShebangs run_no_pty
```

```suggestion
    patchShebangs run_no_pty ./tests/${language}/*.exp
```

this can take an arbitrary number of args, so might as well just call it once.